lib/LaTeXML/Post.pm

```diff
@@ -447,7 +447,7 @@ sub convertXMTextContent {
   foreach my $node (@nodes) {
     if ($node->nodeType == XML_TEXT_NODE) {
       my $string = $node->textContent;
-      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
```

Comment:
On 08/11/2015 08:19 AM, Deyan Ginev wrote:
> In lib/LaTeXML/Post.pm
> <https://github.com/KWARC/LaTeXML/pull/8#discussion_r36738065>:
>
>> @@ -447,7 +447,7 @@ sub convertXMTextContent {
>>    foreach my $node (@nodes) {
>>      if ($node->nodeType == XML_TEXT_NODE) {
>>        my $string = $node->textContent;
>> -      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
>
> Already merged, I think we're up to date:
> https://github.com/KWARC/LaTeXML/commits/master

Confused by the snippet: should be an extra argument to convertXMTextContent
and
   if($convertspaces) {
     $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/; }
without the "should we???" :>


>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KWARC/LaTeXML/pull/8/files#r36738065>.
>